source/default/_patterns/00-protons/legacy/css/global.css

```diff
@@ -138,4 +138,31 @@ div.site-alert div.text {
   padding: 0.3rem;
   margin: 0px;
   font-size: 18px;
+}
+
+/* For use in views where : (colon) is not accepted. */
+
+
+@media (min-width: 640px) {
+  .hide-sm {
+    display: none !important;
+  }
+}
+
+@media (min-width: 768px) {
+  .hide-md {
+    display: none !important;
+  }
+}
+  
+@media (min-width: 1024px) {
+.hide-lg {
+  display: none !important;
+  }
+}
+
+@media (min-width: 1280px) {
+.hide-xl { 
+  display: none !important;
+ }
```

Comment:
@bmartinez287 this is how I am working around the colon escaping issue for the view. I just created a few global classes to allow me to hide fields on various screen sizes. The underscore solution you found might be better, I need to take another look at it.

Reply:
Yeah, the underscore solution is tricky because once I made that change we have to do underscores everywhere even where the colon would not be an issue.